avocado/plugins/replay.py

```diff
@@ -97,10 +97,18 @@ def run(self, args):
 
         view = output.View()
 
-        if args.remote_hostname is not None:
-            msg = "Currently we don't replay jobs in remote hosts."
-            view.notify(event='error', msg=(msg))
-            sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+        err = None
+        if args.replay_teststatus and args.multiplex_files:
+            err = "Option --replay-test-status is incompatible with "\
+                  "--multiplex-files."
+        elif args.replay_teststatus and args.url:
+            err = "Option --replay-test-status is incompatible with "\
+                  "informing new tests in command line."
```

Comment:
We call "tests" in the `avocado run` command line "url":

```
usage: avocado run [-h] [-d] [-z] [--force-job-id UNIQUE_JOB_ID]
   ...
   [url [url ...]]
```

Let's try to use the same terminology. Maybe "test URLs" is even more precise.